### ISSUE TITLE
Tabs inside steps doesn't working properly

### ISSUE THREAD

**Issue by BorMor** [imfing/hextra#437]
Hello, I'm trying to put [tabs](https://imfing.github.io/hextra/docs/guide/shortcodes/tabs/) into a [step](https://imfing.github.io/hextra/docs/guide/shortcodes/steps/) but second tab doesn't work properly

**Steps To Reproduce**
Use this simple code snipped to reproduce the issue
```
{{% steps %}}

### Step one

{{< tabs items="JSON,YAML,TOML" >}}

  {{< tab >}}**JSON**: JavaScript Object Notation (JSON){{< /tab >}}
  {{< tab >}}**YAML**: YAML is a human-readable data serialization language.{{< /tab >}}
  
{{< /tabs >}}

{{% /steps %}}
```

**Expected Behavior**

Second and further tabs show properly

**Actual Behavior**

Second and further tabs not show content and label properly

**Screenshots**

![image](https://github.com/user-attachments/assets/29c62a35-b43a-49ae-9f7a-113de37bc734)

**Environment**

- Hugo Version: v0.132.2
- Browser/OS: FireFox/Windows 10
- Theme Version: main branch [e83c11f](https://github.com/imfing/hextra/tree/e83c11f31a00c1088cd49e8eaea10b60384921c4)

**Additional Context**

**Comment by imfing** [comment]
This will be fixed by #440 

Please be aware that the `Step` shortcode is intended only for Markdown. So it's not recommended to put other shortcodes as content.

**Comment by imfing** [comment]
@joeyundis-altra can you create a new issue?
